src/main/java/duckie/command/ByeCommand.java

```diff
@@ -0,0 +1,30 @@
+package duckie.command;
+
+import duckie.task.TaskList;
+import duckie.Ui;
+import duckie.Storage;
+
+/**
+ * Command to signify the exit of the chatbot
+ */
+public class ByeCommand extends Command {
+    /**
+     * Check if the Command is exiting
+     * @return true
+     */
+    @Override
+    public boolean isExit() {
+        return true;
+    }
+
+    /**
+     * Display the ending message and end the chatbot
+     * @param tasks TaskList containing all the tasks
```

Comment:
Should there be a fullstop behind each param line because "Punctuation behind each parameter description"